scripts/test/Elasticsearch/IndexSetting.t

```diff
@@ -35,48 +37,57 @@ my $MigObject = $Kernel::OM->Get('Kernel::System::Console::Command::Maint::Elast
 
 my $Index1 = $MigObject->_IndexSettingsGet(
     Config => {
-        NS => 5,
-        NR => 10,
+        NS          => 5,
+        NR          => 10,
         FieldsLimit => 1000,
-    });
+    }
+);
 
-$Self->Is(
-    $Index1->{index}->{number_of_shards}, 5,
-    '#1 number_of_shards');
-$Self->Is(
-    $Index1->{index}->{number_of_replicas}, 10,
-    '#1 number_of_replicas');
-$Self->Is(
-    $Index1->{'index.mapping.total_fields.limit'}, 1000,
-    '#1 index.mapping.total_fields.limit');
+Test2::V0::is(
```

Comment:
In test scripts the best practice is to use simply `is()`. E.g.
`is( $index1, { NS => 5, NR => 10, FieldsLimit => 1000 }, 'index1' );`

Besides, I'm confused again. I thought that `_IndexSettingsGet()` should transform the Config to a data structure with keys like `number_of_shards`.

Reply:
That depends on the config settings. As you can see in the expansion tests below in this file it depends on the layout which is set with 'Elasticsearch::IndexSettings'. The first four tests do not use a verbose layout.